ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -27,6 +27,7 @@ module Ouroboros.Consensus.Cardano.Node (
   , protocolInfoCardano
     -- * SupportedNetworkProtocolVersion
   , pattern CardanoNodeToClientVersion1
+  , pattern CardanoNodeToClientVersion10
```

Comment:
Is it `stylish-haskell` imposing lexical order?

Reply:
Yeah

Reply:
The style checker in CI required I do this change.